docsrc/conf.py

```diff
@@ -160,4 +161,17 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-# html_static_path = ['_static']
\ No newline at end of file
+# html_static_path = ['_static']
+
+typehints_use_signature = True
+typehints_use_signature_return = True
+autodoc_typehints = "both"
+autodoc_typehints_format = "short"
+
+#conf.py
+def autodoc_process_docstring(app, what, name, obj, options, lines):
+    for i in range(len(lines)):
+        lines[i] = lines[i].replace("np.", "numpy.")
+
+def setup(app):
+    app.connect("autodoc-process-docstring", autodoc_process_docstring)
```

Comment:
I don't think we should worry about the docstring types in this PR, so I'd leave this out for now.

Reply:
This code is to fix the issue of not linking to `np.ndarray` in docstring. We'll cover it in another PR, so can we just delete it here?

Reply:
sounds reasonable!

Reply:
I fixed it